server/core/pom.xml

```diff
@@ -206,6 +206,33 @@
                </execution>
             </executions>
          </plugin>
+         <!-- We copy the netty BlockHound files into our server core test jar as downstream tools strip these from netty -->
```

Comment:
if downstream strips those files, wouldn't this step fail? It would try to unpack from the "downstream" jar file.

Reply:
I was unaware of that.. I think this has to go back to the drawing board then :(